csm/conf/setup.py

```diff
@@ -512,29 +472,47 @@ def _set_rmq_cluster_nodes(self):
 
             raise CsmSetupError(f"Setting RMQ cluster nodes failed. {e} - {str(traceback.print_exc())}")
 
-    def _set_consul_vip(self):
+    @staticmethod
+    def _get_minion_id():
         """
-        This method gets the consul VIP for the current node and sets the
-        value in database.yaml config.
-        Seting the default values in case of command failure.
+        Obtains current minion id. If it cannot be obtained, returns default node #1 id.
         """
-        minion_id = None
-        consul_host = None
+        minion_id = SaltWrappers.get_salt_call(const.GRAINS_GET, const.ID)
+        if not minion_id:
+            Log.logger.warn(
+                f'Could not fetch current minion id; using {const.MINION_NODE1_ID} instead')
+            minion_id = const.MINION_NODE1_ID
```

Comment:
Setup should be failed, if minion id is not found.

Reply:
This is just a rewrite of the already existing workflow for the Consul backend. Please check https://github.com/Seagate/cortx-manager/pull/248/files#diff-c79c4c328e23b4882fbe2daee6d10a61597210e0e9fb7940de66702409de38e6L525

I did not remove it because I didn't want to risk radically changing an existing functionality in a release branch.

Reply:
Fixed. Now we raise an error in case the minion id could not be obtained.